.cirrus.yml

```
@@ -3,8 +3,9 @@ container:
   cpu: 1
   memory: 1G
 lint_task:
-  use_compute_credits: true
-  only_if: $CIRRUS_BASE_BRANCH == 'main'
+  use_compute_credits: true  # https://cirrus-ci.org/pricing/#compute-credits
+  # https://cirrus-ci.org/guide/writing-tasks/#conditional-task-execution: Skip forks and non-pulls, to skip needless compute
+  skip: $CIRRUS_REPO_OWNER != 'bitcoin-core' || $CIRRUS_PR == ""
```

> It seems like an edge case either way, so happy to close this if you don't think it is worth it.

Skipping task for non-pulls is still quite reasonable though:
```suggestion
  skip: $CIRRUS_PR == ""
```